frontend-opal/src/reducers/Watchlists.js

```diff
@@ -1,5 +1,7 @@
 const pairs = {
```

Comment:
@hondahmd name of the file should start with capital if it's a class or component, this is neither